### PR TITLE
Pattern Printing

### PR DESCRIPTION
**Pull request by shubham1617**
Testing pull request

### CODE REVIEW

File: Pattern.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Pattern {
+    public static void main(String[] args) {
+        System.out.println("Pattern Printing...");
```
Comment (ankitskvmdam) [inline Pattern.java]: Could you ignore using hard coded string? It would be better to use a constant.

Comment (shubham1617) [reply Pattern.java:3]: Can't you do it from your side, i think you are too experiance and will make this small changes right ?

Comment (shubham1617) [reply Pattern.java]: Sure

Comment (ankitskvmdam) [reply Pattern.java]: Looks good!

**Review by ankitskvmdam — APPROVED** [review]
LGTM!

**Pull request merged**